docs/helpers/StorageFiles.md

```diff
@@ -22,6 +23,12 @@ The StorageFileHelper is a static utility class that provides functions to help
 	// Load some text from a file named appFilename.txt in the local folder	
 	loadedText = await StorageFileHelper.ReadTextFromLocalFileAsync("appFilename.txt");
 
+	// Check if a file exists in a specific folder
+	bool exists = await StorageFileHelper.FileExistsAsync(localFolder, "appFilename.txt");
```

Comment:
Should we demonstrate this version or the extension? (I'm more for the extension)